packages/mobile/src/transactions/transferFeedUtils.ts

```diff
@@ -224,7 +231,7 @@ export function getTransferFeedParams(
   return { title, info, recipient }
 }
 
-export function getTxsFromUserTxQuery(data: UserTransactionsQuery | undefined) {
+export function getTxsFromUserTxQuery(data?: UserTransactionsQuery) {
```

Comment:
Why this change?
Will be functionally the same at runtime, but semantically it's different, and will change the TypeScript check.
The previous one required a parameter to be passed (even if undefined), the changed version makes the parameter optional.
In this case, expecting data to be passed seems what we want.